test/evergreen/evg_cfg.py

```diff
@@ -112,14 +115,15 @@ def find_tests_missing_evg_cfg(test_type, dirs, evg_cfg_file):
         else:
             sys.exit("Unsupported test_type '%s'" % test_type)
 
-        # Check if the Evergreen task name exists in current Evergreen configuration
-        if evg_task_name in str(evg_cfg):
+        # Check if the Evergreen task name exists in current Evergreen configuration or in the
+        # csuite CMakeLists file.
+        if evg_task_name in str(evg_cfg) or d in str(cmake_cfg):
```

Comment:
As reflected by the name, this function aims at finding those missing tests in the Evergreen config file. This added sidecar check with a slightly different purpose of identifying missing tests in the cmake list file - which is less likely to happen. 

It does feel with the merge of individual csuite tasks there's no need to check missing tests for the "csuite" test_type - meaning the relevant code block can be removed. All newly introduced csuite tests in the future (in cmake list file) will be automatically covered by the new combined csuite task.

Reply:
hmm I agree, but it is also possible that we forget to add the csuite test into the CMakeList. @lukech what do you think should be done here?

Reply:
> it is also possible that we forget to add the csuite test into the CMakeList

It's possible, and the risk was there before this PR. If we really want to introduce the check (i.e. making sure new csuite dirs/tests are added in the cmake list) I'd suggest using a separate ticket/PR and not putting the check in `evg_cfg.py`. It feels like the risk of missing an entry in cmake list is relatively lower than missing a test in Evergreen config file, so I'm too worried about that. 

So my proposal is to remove the newly added cmake list check, and the previous "csuite" test_type checks from `evg_cfg.py`.